reinforce/training/trainer.py

```
@@ -0,0 +1,459 @@
+            while self._state.step < end_step:
+                step = self._state.step
+
+                # ##>: Generate new games periodically using parallel execution.
+                if step > 0 and step % self.config.generation_interval == 0:
+                    self._generate_games_parallel()
+
+                # ##>: Sample batch and train.
+                if self.use_async_loader and self._async_loader is not None:
+                    batch, weights = self._async_loader.get_batch()
+                else:
+                    batch, weights = self._buffer.sample_batch_vectorized(self.config.batch_size)
```

Ruff (0.14.10)

Unpacked variable `weights` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)